didact/src/components/courses/EditCourse.jsx

```diff
@@ -107,12 +107,15 @@ const useStyles = makeStyles(theme => ({
 
     descriptionDiv: {
         display: "flex",
-        flexDirection: "row",
-        justifyContent: 'center',
+        width: "100%",
+        flexDirection: 'column',
+        justifyContent: "space-between",
+        fontSize: 12,
+        color: "#757575"
```

Comment:
We should be using color variables somewhere, and importing them in style files.